### ISSUE TITLE
Nimbus - could not find `process_resident_memory_bytes` in metrics

### ISSUE THREAD

**Issue by joebiker** [Alethio/eth2stats-client#70]
Running Nimbus client 0.6.6 
Running eth2stats-client version v0.0.16+d729a1d

Description of warn message
stats is looking for `process_resident_memory_bytes`

Closest stat that Nimbus reports is
nim_gc_mem_bytes 4100096.0
nim_gc_mem_occupied_bytes 2733144.0
sqlite3_memory_used_bytes 2614336.0

**Comment by protolambda** [comment]
Have you tried asking in the discord? I think it should work if you use the correct metrics flags (see readme of this repo for help), and nimbus might have to be built with the "insecure" flag set to true (I think it was `NIMFLAGS="-d:insecure"`) to enable extra HTTP functionality like metrics (this might have changed, ask Nimbus team).

**Comment by joebiker** [comment]
@protolambda Yes, I've done that. I compiled using the `NIMFLAGS` suggested.
I would revert back to nimbus 0.6.5 to show the difference in stats, but I'm struggling to get eth2stats to run as a daemon on mac launchctl (exits 1 everytime).

**Comment by joebiker** [comment]
I see this in the README.md
`The process_resident_memory_bytes gauge is extracted from the Prometheus metrics endpoint.`

However I'm not sure how it applies. I am running Prometheus, however there's no configuration for including Prometheus into command line options?

**Comment by protolambda** [comment]
That's poor wording in the readme, it should just grab it from the metrics endpoint of the eth2 client. Prometheus is the more common consumer of the metrics endpoint, but not relevant here.

Are you using all the nimbus metrics flags? `--metrics --metrics-port=8080 --metrics-address=0.0.0.0`
You could try `curl` the `/metrics` endpoint to check if it can be reached and has the expected contents.

**Comment by joebiker** [comment]
Yes, I checked the `/metrics` endpoint on Nimbus 0.6.6, I've listed the contents of what I consider 'close' to the memory usage statistic.
